### PR TITLE
Eigen free

### PR DESCRIPTION
**Pull request by bptigg**

### CODE REVIEW

File: RunSection/Action.cpp
Comment (Das0Mann) [inline RunSection/Action.cpp:1]: is this still required?

File: RunSection/Action.h
Comment (Das0Mann) [inline RunSection/Action.h:1]: I think the parellelize is not required here

File: RunSection/RunSection.h
Comment (Das0Mann) [inline RunSection/RunSection.h:1]: is this required? I mean a Getfucntion for the action objects is not wrong, but its not used isnt it?

File: RunSection/RunSection.cpp
Comment (Das0Mann) [inline RunSection/RunSection.cpp:1]: oh was this an issue before?

**Review by Das0Mann — CHANGES_REQUESTED** [review]
Some minor questions

File: RunSection/RunSection.cpp
Comment (bptigg) [reply RunSection/RunSection.cpp:1]: yeah, I only noticed it when I had actions for scalars, i.e it tried to find the state SingletDecay_Wc

File: RunSection/RunSection.h
Comment (bptigg) [reply RunSection/RunSection.h:1]: I think it was used at somepoint but no longer is when I got rid of some of the parallel code

**Review by bptigg — COMMENTED** [review]
I'll upload the amended code when I get to uni, train wifi is being a bit funky

File: RunSection/RunSection.cpp
Comment (Das0Mann) [reply RunSection/RunSection.cpp:1]: very nice thx!

**Review by Das0Mann — APPROVED** [review]

**Pull request merged**